data/bhavanas-jpg.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Bhavana S",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Frontend developer | Blogger | Poetry & Literature lover| Yoga Practitioner",
+  "avatar": "https://pbs.twimg.com/profile_images/1607335763548725249/KDIKS-wp_400x400.jpg",
```

Comment:
You can copy your GitHub profile URL and just add `.png` at the end.

Reply:
Thank you for the suggestion, Abbey. People can have a custom link. Thank you.

Reply:
Thanks for this Pradumna. Gonna take note for future PRs.